src/mumble/ServerHandler.cpp

```diff
@@ -100,14 +100,32 @@ ServerHandler::ServerHandler() {
 	MumbleSSL::addSystemCA();
 
 	{
+		QString cipherString = g.s.qsSslCiphers.trimmed();
+		if (cipherString.isEmpty()) {
+			cipherString = MumbleSSL::defaultOpenSSLCipherString();
```

Comment:
Why not set the default value of the config option to defaultOpenSSLCipherString() and remove this special handling? Too early in startup?